packages/bun/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@sentry/bun",
+  "version": "7.69.0",
+  "description": "Official Sentry SDK for bun",
+  "repository": "git://github.com/getsentry/sentry-javascript.git",
+  "homepage": "https://github.com/getsentry/sentry-javascript/tree/master/packages/bun",
+  "author": "Sentry",
+  "license": "MIT",
+  "engines": {
+    "node": ">=8"
+  },
+  "main": "build/cjs/index.js",
+  "module": "build/esm/index.js",
+  "types": "build/types/index.d.ts",
+  "typesVersions": {
+    "<4.9": {
+      "build/npm/types/index.d.ts": [
+        "build/npm/types-ts3.8/index.d.ts"
+      ]
+    }
+  },
+  "publishConfig": {
+    "access": "public"
+  },
```

Comment:
This is fine (no action required) but just to note: With this PR we will not yet publish the Bun SDK because it's not configured in `craft.yml`. We can tackle this in a follow-up PR!